ios/browser/flags/about_flags.mm

```diff
@@ -74,16 +75,25 @@
                                  kBraveWalletTransactionSimulationsFeature),  \
       })
 
-#define BRAVE_SHIELDS_FEATURE_ENTRIES                                    \
-  EXPAND_FEATURE_ENTRIES({                                               \
-      "brave-shred",                                                     \
-      "Enable Brave 'Shred' Feature",                                    \
-      "Enable the Brave ‘Shred’ feature which will allow a user to " \
-      "easily delete all site data on demand or automatically when "     \
-      "closing a site or terminating the application.",                  \
-      flags_ui::kOsIos,                                                  \
-      FEATURE_VALUE_TYPE(brave_shields::features::kBraveShredFeature),   \
-  })
+#define BRAVE_SHIELDS_FEATURE_ENTRIES                                        \
+  EXPAND_FEATURE_ENTRIES(                                                    \
+      {                                                                      \
+          "brave-shred",                                                     \
+          "Enable Brave 'Shred' Feature",                                    \
+          "Enable the Brave ‘Shred’ feature which will allow a user to " \
+          "easily delete all site data on demand or automatically when "     \
+          "closing a site or terminating the application.",                  \
+          flags_ui::kOsIos,                                                  \
+          FEATURE_VALUE_TYPE(brave_shields::features::kBraveShredFeature),   \
+      },                                                                     \
+      {                                                                      \
+          "https-by-default",                                                \
```

Comment:
I see an "HTTPS Upgrades" feature flag which is default disabled on iOS, how does that relate to this one?

Reply:
These control the similar Chromium iOS features, which are only supported in Chromium web views https://source.chromium.org/chromium/chromium/src/+/main:ios/components/security_interstitials/https_only_mode/feature.h

The goal will be to use these in the future either in this PR https://github.com/brave/brave-core/pull/24657 or after it merges